addOns/soap/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 ### Changed
 - Maintenance changes.
+- WSDL File passive scanner now includes identification of URLs in the form "example.com/service?wsdl", and specifically excludes 404s and 500s (including support for Custom Pages).
+- `Import a WSDL file from a URL` now logs a more specific message when a WrongGrammarException is encountered, or when failing to convert the input 'URL' string into an actual URL to make the WSDL request.
```

Comment:
Not sure the name of the exception is useful (I'd have to check the JavaDoc/code to know for sure what it is about).